ledger/internal/cow.go

```diff
@@ -123,7 +123,9 @@ func makeRoundCowState(b roundCowParent, hdr bookkeeping.BlockHeader, proto conf
 	return &cb
 }
 
-func (cb *roundCowState) deltas() ledgercore.StateDelta {
+// Updates returns a ledgercore.StateDelta containing all the changes made to this roundCowState.
+// However, it does not return old state information (i.e. cb.mods.KvMods[key].OldValue).
+func (cb *roundCowState) Updates() ledgercore.StateDelta {
```

Comment:
so the take from what I learnt this morning is that, `Updates` stand for only new variables, but we are not interested in old values, thus `deltas` implementation retain the old variable collections.